specification/components/examples/requests/NHAIS/acceptance-birth-max.json

```diff
@@ -0,0 +1,66 @@
+{
+  "id": "54321",
+  "extension": [
+    {
+      "url": "http://hl7.org/fhir/StructureDefinition/patient-birthPlace",
+      "valueString": "TownOfBirth"
+    },
+    {
+      "url": "https://fhir.nhs.uk/R4/StructureDefinition/Extension-UKCore-NHAIS-DrugsDispensedMarker",
+      "valueBoolean": true
+    },
+    {
+      "url": "https://fhir.nhs.uk/R4/StructureDefinition/Extension-UKCore-NHAIS-ResidentialInstituteCode",
+      "valueString": "ResidentialInstituteCode"
+    },
+    {
+      "url": "https://fhir.nhs.uk/R4/StructureDefinition/Extension-UKCore-NHAIS-GPNotes",
+      "valueString": "GP Notes Free Text"
+    }
+  ],
+  "address": [
+    {
+      "line": [
+        "current house name",
+        "current number or road name",
+        "current locality",
+        "current town",
+        "current county"
+      ],
+      "postalCode": "SO15 2GB"
+    }
+  ],
+  "birthDate": "1990-01-01T09:33:00",
```

Comment:
should max birth date be 1990?

Reply:
max is an maximum number of fields (including all optional) not maximum values